### PR TITLE
Caters for profiles having multiple label & desscription predicates

### PR DESCRIPTION
**Pull request by jamiefeiss**
This small update should fix the labeling issues when a profile has multiple altr-ext:hasLabelPredicate & altr-ext:hasDescriptionPredicate predicates.

### CODE REVIEW

File: src/views/ItemListView.vue [hunk 1]
```diff
@@ -121,19 +121,19 @@ onMounted(() => {
...
-            }
-            if (currentProfile.descPredicate) {
-                descPred = currentProfile.descPredicate;
-            }
+            if (currentProfile.labelPredicate.length > 0) {
+            labelPred = currentProfile.labelPredicate;
+        }
+        if (currentProfile.descPredicate.length > 0) {
+            descPred = currentProfile.descPredicate;
+        }
```
Comment (edmondchuc) [inline src/views/ItemListView.vue:136]: Indentation

File: src/types.ts [hunk 2]
```diff
@@ -49,8 +49,8 @@ export interface Profile {
     description: string;
     mediatypes: string[];
     defaultMediatype: string;
-    labelPredicate?: string;
-    descPredicate?: string;
+    labelPredicate: string[];
+    descPredicate: string[];
```
Comment (edmondchuc) [inline src/types.ts:53]: Now that it's an array for these two fields, can we change it to plural form?

**Review by edmondchuc — APPROVED** [review]
Besides some small cosmetic changes, looks good to me.

**Comment by recalcitrantsupplant** [comment]
there were three annotation properties, with defaults as below: 

    label_predicates += [RDFS.label]
    description_predicates += [DCTERMS.description]
    explanation_predicates += [DCTERMS.provenance]

I think Nick or you requested the third one - provenance - and I have it down as an "explanation" property - is it in use by the frontend and if not, will it be or should I remove it

**Comment by jamiefeiss** [comment]
@recalcitrantsupplant explanation/provenance is used for annotations for predicates, etc. Would explanation be used on objects like spatial datasets for display purposes? Happy to include explanation in the frontend anyway if there's any uncertainty.

File: src/types.ts (hunk 2 above)
Comment (jamiefeiss) [reply src/types.ts:53]: Yes I agree, plural form makes more sense

**Comment by recalcitrantsupplant** [comment]
Not sure - is there an example API up somewhere that uses these? @nicholascar

File: src/types.ts (hunk 2 above)
Comment (jamiefeiss) [reply src/types.ts]: Resolved in latest commit

File: src/views/ItemListView.vue (hunk 1 above)
Comment (jamiefeiss) [reply src/views/ItemListView.vue]: Resolved in latest commit

**Comment by nicholascar** [comment]
> I think Nick or you requested the third one - provenance - and I have it down as an "explanation" property - is it in use by the frontend and if not, will it be or should I remove it

> is there an example API up somewhere that uses these?

Well there was _meant_ to be one and it was to be the GA Samples API that you made @recalcitrantsupplant! The idea was that the reason for using a particular property could be revealed in the UI.

Just don't worry about this for now: let's merge this PR and come back to provenance for properties if we encounter the need again.

**Review by edmondchuc — APPROVED** [review]

**Comment by jamiefeiss** [comment]
Resolved merge conflicts and fixed a TypeScript error. Should be all good to merge now.

**Review by edmondchuc — APPROVED** [review]

**Pull request merged**